.gitignore

```diff
@@ -10,3 +10,5 @@ tmp
 .classpath
 .project
 .settings
+*.swp
+tags
```

Comment:
> So that's an argument for allowing all files in here that any contributor wants to ignore. 

My 2 cents:

:+1: I agree with this philosophy. It's also the way the front-end repos work. If someone made a change and wanted to stop ignoring certain types of files, because for some reason files with that name / structure became important to the project, we'd want it to be reflected across the whole development team, which wouldn't work if every developer kept such changes in their own private `~/.config/git/ignore`.